tests/test_integrator/test_fiat_adapter.py

```diff
@@ -183,3 +185,25 @@ def test_fiat_adapter_measures():
         exposure_scenario.loc[inds2, "Damage Function: Structure"]
         != exposure_template.loc[inds1, "Damage Function: Structure"]
     )
+
+
+def test_fiat_return_periods():
+    test_toml = (
+        test_database
+        / "charleston"
+        / "input"
+        / "scenarios"
+        / "current_extreme12ft_no_measures_return_periods"
+        / "current_extreme12ft_no_measures_return_periods.toml"
+    )
+
+    assert test_toml.is_file()
+
+    # use event template to get the associated Event child class
+    test_scenario = Scenario.load_file(test_toml)
+    test_scenario.init_object_model()
+    # TODO: Hazard class should check if the hazard simulation has already been run when initialized
+    test_scenario.direct_impacts.hazard.has_run = True  # manually change this for now
+    test_scenario.direct_impacts.hazard.event_mode = "risk"
+    test_scenario.direct_impacts.hazard.set_sfincs_map_path(mode="risk")
```

Comment:
These three manual changes are not needed. If the models has run it should be found automatically.